sitemedia/admin/js/person-editlink.js

```diff
@@ -0,0 +1,49 @@
+/*
+Customize person edit input (dal select2) on account page to generate
+a link to the person edit page based on the selection.
+*/
+$(document).ready(function() {
+    // store reference to the form input with the person ids
+    var person_select = $('#id_persons');
+    // create container for displaying edit links
+    var editlinks = $('<div/>').attr('id', 'person-editlink').text('test');
+    person_select.parent().append(editlinks);
+
+    // function to update person edit links
+    function display_edit_links() {
+        // remove any current/outdated edit links
+        editlinks.empty();
+        var person_ids = person_select.val();
+
+        // add a new link for each current value
+        $(person_select.val()).each(function(idx, person_id) {
+            var para, link;
+            // create a link within a paragraph and add to the edit links div
+            link = $('<a>')
+                .attr('target', '_blank')   // open in new tab
+                // FIXME: how to make this not hard-coded? Can we generate
+                // a template url in the template and replace the id?
+                .attr('href', '/admin/people/person/' + person_id + '/change/')
```

Comment:
Re: the `FIXME`. A thought occurs: create a hidden div in a template override that holds the link generated using `{% url %}` and a default `pk` of `1` that could be used as a template for a regex replace of some sort.

I am not sure how much of an improvement that would be. It gives us flexibility for a changed route and avoids the hardcoding, but at the cost of added complexity and tweaking the template further. The trade off seems close to me.